Module not found: @babel/runtime

**Describe the bug**
I'm getting an error after updating NextJS version:

```ShellSession
error - ./node_modules/next-firebase-auth/build/index.browser.js:1:1496
Module not found: Can't resolve '@babel/runtime/helpers/objectWithoutProperties'

Import trace for requested module:
./services/initAuth.js
./pages/_app.js
```

**Versions**

`next-firebase-auth` version: 1.0.0-canary.4
Firebase JS SDK:
- firebase: 9.6.1
- firebase-admin: 10.0.1

Next.js: 12.0.8-canary.13 (updated from 12.0.7)

**To Reproduce**
Steps to reproduce the behavior:
1. Create a project following [your guide](https://github.com/gladly-team/next-firebase-auth#get-started)
2. Use next 12.0.8-canary.13
3. See error

**Expected behavior**
Should work, but fails.

**Additional context**
It seems NextJS is [pre-compiling](https://github.com/vercel/next.js/pull/32679) its dependencies and now @babel/runtime has been precompiled so it is not available and must be set as dependency in your package.
https://babeljs.io/docs/en/babel-plugin-transform-runtime

**Temporary fix**
Install the package as a dependency in the webpage project: `yarn add @babel/runtime` or `npm i @babel/runtime` (then it works).

Thank you very much for this great package, it is really useful 📈🚀

--- a/package.json
+++ b/package.json
@@ -107,6 +107,7 @@
     }
   },
   "dependencies": {
+    "@babel/runtime": "^7.18.9",
     "cookies": "^0.8.0",
     "hoist-non-react-statics": "^3.3.2"
   },

